src/VisualStudio/Core/Def/Implementation/Options/LanguageSettingsPersister.cs

```diff
@@ -56,9 +56,12 @@ internal sealed class LanguageSettingsPersister : ForegroundThreadAffinitizedObj
             {
```

Comment:
regarding not knowing about TypeScript's guid.  Why can't we have AbstractPackage (or whatever) register some hook into this.  That way we'll know about all these langauges and guids in a dynamic fashion instead of needing to hardcode it in?